src/services/delete-todo.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from '@nestjs/common';
+import { CacheClear } from '@type-cacheable/core';
+import { todoCacheKey } from '@/consts/todo';
+import { TodoRepository } from '@/repositories/todo.repository';
+@Injectable()
+export class DeleteTodoService {
+  constructor(private todoRepository: TodoRepository) {}
+  @CacheClear({ cacheKey: ([id]: [string]) => [id, todoCacheKey] })
+  async delete(id: string): Promise<true> {
+    const todo = await this.todoRepository.findOneOrFail(id);
+    await this.todoRepository.remove(todo);
```

Comment:
Do you know if todoRepository.remove returns true by default? If so, it is not necessary the next line.

Reply:
No :(